<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcQHTMLDialog</class>
 <widget class="QDialog" name="lcQHTMLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HTML Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Output folder:</string>
          </property>
          <property name="buddy">
           <cstring>outputFolder</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputFolder"/>
        </item>
        <item>
         <widget class="QToolButton" name="outputFolderBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="transparentImages">
        <property name="text">
         <string>Transparent image background</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Models</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="allModels">
        <property name="text">
         <string>All Models</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="currentModelSubmodels">
        <property name="text">
         <string>Current Model and Submodels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="currentModelOnly">
        <property name="text">
         <string>Current Model Only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Steps</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="singlePage">
        <property name="text">
         <string>Single Page</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="oneStepPerPage">
        <property name="text">
         <string>One Step per Page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="indexPage">
        <property name="text">
         <string>Index page</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="buddy">
         <cstring>stepWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="stepWidth">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>stepHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="stepHeight">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parts List</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="partsAtTheEnd">
        <property name="text">
         <string>At the end</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="partsAfterEachStep">
        <property name="text">
         <string>After each step</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outputFolder</tabstop>
  <tabstop>outputFolderBrowse</tabstop>
  <tabstop>transparentImages</tabstop>
  <tabstop>allModels</tabstop>
  <tabstop>currentModelSubmodels</tabstop>
  <tabstop>currentModelOnly</tabstop>
  <tabstop>singlePage</tabstop>
  <tabstop>oneStepPerPage</tabstop>
  <tabstop>indexPage</tabstop>
  <tabstop>stepWidth</tabstop>
  <tabstop>stepHeight</tabstop>
  <tabstop>partsAtTheEnd</tabstop>
  <tabstop>partsAfterEachStep</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcQHTMLDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcQHTMLDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
